content/_data/upgrades/2-303-1.adoc

```diff
@@ -0,0 +1,176 @@
+* `jenkins/ssh-agent:alpine-jdk8`
+* `jenkins/ssh-agent:latest-jdk8`
+* `jenkins/ssh-agent:3.1.0-jdk8`
+* `jenkins/ssh-agent:3.1.0-alpine-jdk8`
+* `jenkins/ssh-agent:3.1.0-windowsservercore-ltsc2019-jdk8`
+
+===== Agent images
+
+Jenkins agent images that do not specify a Java version now use Java 11 instead of Java 8.
+Jenkins agent images with the following tags will use Java 11:
+
+* `jenkins/agent:alpine`
+* `jenkins/agent:archlinux`
+* `jenkins/agent:latest`
+* `jenkins/agent:latest-archlinux`
+* `jenkins/agent:bullseye`
+* `jenkins/agent:4.10-1`
+* `jenkins/agent:4.10-1-alpine`
+* `jenkins/agent:4.10-1-windowsservercore-ltsc2019`
+
+Additional agent images and tags have been added for those users who need to remain on Java 8.
+The new Java 8 tags include:
+
+* `jenkins/agent:alpine-jdk8`
+* `jenkins/agent:archlinux-jdk8`
+* `jenkins/agent:latest-jdk8`
+* `jenkins/agent:latest-archlinux-jdk8`
+* `jenkins/agent:bullseye-jdk8`
+* `jenkins/agent:4.10-1-jdk8`
+* `jenkins/agent:4.10-1-alpine-jdk8`
+* `jenkins/agent:4.10-1-windowsservercore-ltsc2019-jdk8`
+
+==== Obsolete Docker images and tags no longer published
+
+Jenkins Docker images for new releases are not published for `centos`.
+The `centos` images use CentOS 8 as a baseline and CentOS 8 is no longer receiving updates from the Red Hat Enterprise Linux upstream.
+
+Jenkins Docker Windows images for new releases are not published for the `1809` images.
+Microsoft has ended mainstream support for `1809`.
+
+Jenkins Docker tags that include a version number are no longer published with the redundant `-lts` tag.
+The transition looks like this:
+
+* 2.303.1-lts is replaced with 2.303.1
+* 2.303.1-lts-alpine is replaced with 2.303.1-alpine
+* 2.303.1-lts-centos7 is replaced with 2.303.1-centos7
+* 2.303.1-lts-slim is replaced with 2.303.1-slim
+
+==== Digester 2 removed
+
+The Apache Commons Digester v2.1 has been removed from Jenkins core.
+The library was last released in 2010.
+Plugins that depend on the library need to be updated to remove that dependency.
+
+Plugins that are affected include:
+
+* link:https://plugins.jenkins.io/vs-code-metrics/[Visual Studio Code Metrics]
+* link:https://plugins.jenkins.io/BlameSubversion/[Blame Subversion]
+* link:https://plugins.jenkins.io/javatest-report/[JavaTest report]
+* link:https://plugins.jenkins.io/vss/[Visual SourceSafe]
+* link:https://plugins.jenkins.io/synergy/[Synergy]
+* link:https://plugins.jenkins.io/config-rotator/[Config Rotator]
+* link:https://plugins.jenkins.io/harvest/[Harvest SCM]
+* link:https://plugins.jenkins.io/cmvc/[CMVC]
+
+Suspended plugins (no longer distributed by the Jenkins update center) that are affected include:
+
+* link:https://plugins.jenkins.io/tfs/[TFS]
+* link:https://issues.jenkins-ci.org/browse/INFRA-2487[svn-release-mgr]
+* link:https://plugins.jenkins.io/cpptest/[cpptest]
+* link:https://plugins.jenkins.io/CFLint/[CFLint]
+
+==== Stop bundling External Monitor Job Type, LDAP, and PAM plugins with Jenkins
```

Comment:
Move to bottom after removal?

Reply:
I'm not sure I understand your intent with "move to bottom after removal".  

Are you recommending that this section should be the last section in the upgrade guide?   I'm OK with that, but the "Stop bundling CVS plugin" was not the last section in the 2.277.1 upgrade guide, though the unbundling announcements were the last in the 2.204.1 upgrade guide and in the 2.289.1 upgrade guide.

I assumed this was a higher risk than the other removals that follow it in the upgrade guide.  Did I misunderstand the risk or is there something different you mean by "move to bottom after removal"?